EyeDropper/explainer.md

```diff
@@ -40,55 +39,31 @@ Additionally, browsers should provide a clear indication as to when the user has
 The transition into eyedropper mode should require [consumable user activation](https://github.com/mustaqahmed/user-activation-v2), for example, clicking on a button from the web page, to help avoid unintentionally revealing pixel data.
 
 ## Solution
-The API will enable web developers to incorporate an eyedropper in their web applications. The eyedropper would allow the developer to access the hex value (of the form `#RRGGBB`) of a user specified pixel, its position and modifier keys pressed when the pixel was selected.
-
-The position of the selected color is included to facilitate scenarios where a web app using the eyedropper samples a pixel color from its own document.  The web app could, for example, include an alpha channel for the selected pixel or create a palette of colors associated with a pixel's location based on layer information known to the web app.  The color value would otherwise be the final composited color as seen by the user.
+The API will enable web developers to incorporate an eyedropper in their web applications. The eyedropper would allow the developer to access the hex value (of the form `#RRGGBB`) of a user specified pixel.
```

Comment:
Future versions of the API are expected to expose a more robust representation of color based on the work of other working groups, e.g. (insert link).